packages/slate-editor/src/extensions/placeholders/elements/SocialPostPlaceholderElement.tsx

```diff
@@ -136,6 +136,17 @@ export function SocialPostPlaceholderElement({
         });
     }
 
+    if (element.provider === PlaceholderNode.Provider.X) {
+        return render({
+            inputTitle: 'X post',
+            inputDescription: 'Paste a social media link and hit Enter',
+            inputPlaceholder: 'https://www.x.com/post/',
```

Comment:
I believe they haven't yet migrated the URLs to the new domain. So I suggest to keep it as `twitter.com/post/` to not confuse people. 

@alpower agree?

Reply:
Good call 👍

Used "twitter" instead: https://github.com/prezly/slate/pull/475/commits/0f7a3c9db2ed7c58577edec64e301eb1293636bf